packages/jest-circus-allure-reporter/README.md

```diff
@@ -0,0 +1,152 @@
+# Jest Circus Allure Reporter
```

Comment:
README is simplified from the [original](https://github.com/ryparker/jest-circus-allure-environment/blob/main/README.md):
- removed quick start guide
- removed advanced configuration section
- removed TMS and Jira specifics